lib/command/apply_template.rb

```diff
@@ -8,7 +8,8 @@ class ApplyTemplate < Base # rubocop:disable Metrics/ClassLength
     OPTIONS = [
       app_option(required: true),
       location_option,
-      skip_confirm_option
+      skip_confirm_option,
+      add_app_identity_option
```

Comment:
Added this option because we needed to:

1. Check if there's an identity template
2. If not, add an identity template **right after** the app template - this is important since a) we can't create the identity at the beginning because the app doesn't exist yet, and b) we also can't create it at the end because any workload templates associated with it will fail to apply

That's what the `add_app_identity_template` method does.

So we needed a way for `setup-app` to communicate to `apply-template` that it should do that, and an option was the easiest way to do it.